seatunnel-apis/seatunnel-api-flink/src/main/java/org/apache/seatunnel/flink/stream/FlinkStreamSink.java

```diff
@@ -21,17 +21,13 @@
 import org.apache.seatunnel.flink.FlinkEnvironment;
 
 import org.apache.flink.streaming.api.datastream.DataStream;
-import org.apache.flink.streaming.api.datastream.DataStreamSink;
 import org.apache.flink.types.Row;
 
-import javax.annotation.Nullable;
-
 /**
  * a FlinkStreamSink plugin will write data to other system using Flink DataStream API.
  */
 public interface FlinkStreamSink extends BaseFlinkSink {
 
-    @Nullable
-    DataStreamSink<Row> outputStream(FlinkEnvironment env, DataStream<Row> dataStream);
+    void outputStream(FlinkEnvironment env, DataStream<Row> dataStream);
```

Comment:
Could we remove the FlinkStreamSink and  move the method to BaseFlinkSink? BaseFlinkSource is similar.

Because we don't distinguish the batch or stream, we can just keep the BaseFlinkSink. All flink sinks implement the BaseFlinkSink. How do you think?